Tests/SwiftBasicFormatTest/BasicFormatTests.swift

```diff
@@ -588,6 +588,7 @@ final class BasicFormatTest: XCTestCase {
         func test() {
             Task {
             }
+        
```

Comment:
@ahoppen after your PR was merged, it introduced a newline here.

It is happening inside the `transferTrailingTrivaFromEndOfFileIfPresent`, where we add `endOfFileToken.leadingTriviaPieces`

Reply:
Yes, that’s expected because we no longer discard the `\n\n` after the `Task {\n}`.